src/images/logo/blib-blue-vec.svg

```diff
@@ -0,0 +1 @@
+<svg id="geschlossenes_Element" data-name="geschlossenes Element" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 838.67 513.77"><defs><style>.cls-1{fill:#60a1da;}</style></defs><path class="cls-1" d="M721.31,329.36a60.66,60.66,0,0,1-117.1,22.24l.53,2.51-118.1,23.37a67.73,67.73,0,0,1-135.4.17l-119-23.54a60.68,60.68,0,1,1,5.27-24.75l117.3,24.14h0a67.83,67.83,0,0,1,51.64-44.64h-.06V238.61a60.68,60.68,0,1,1,24.25.15v70a68.21,68.21,0,0,1,52.27,44.61l116.53-24,.6,2.81c-.05-.93-.07-1.87-.07-2.81a60.66,60.66,0,1,1,121.31,0Z"/><path class="cls-1" d="M742,173.51c-55.32-25.88-106.5-23-147.65-6.18-2-42.67-20.88-83.76-57-118C467.11-17.18,353.86-11,287.55,62.75c-27.8,30.93-42,67.41-44.2,104.56-41.13-16.84-92.3-19.67-147.59,6.2C25.52,206.37-29.84,309.63,20.69,409.65c36.82,72.9,128.49,107.57,187,101.64l423.57.13C689.81,516.73,780.46,482,817,409.65,867.55,309.63,812.19,206.37,742,173.51Zm14.59,209.23c-21.55,39.7-58.75,54.73-100.77,58.54H506.59c-19.39,24.45-47.34,43.58-87.55,43.58-35.83,0-66.42-15.25-87.55-43.58l-148,.14c-42.66-3.58-80.52-18.49-102.34-58.68-31.84-58.66-6.58-125.45,48.37-152.32s122-1.85,148,54.67a114.28,114.28,0,0,1,6.17,18.16s4.23-5,4.93-5.77c20.46-21.57,33.53-29,53.32-39.92a91.69,91.69,0,0,1-13.78-16.36c-.57-1-1-1.81-1.23-2.12-23.16-34.32-26.71-89.82,9.24-131.84C375.52,61.2,442.68,56.93,490,95.66c31.88,26.08,52.19,79.56,27.63,131.8l-.07.1s-8.54,17.93-21.78,30c19.79,11,32.86,18.35,53.32,39.92.7.74,4.88,5.73,4.93,5.77a115.3,115.3,0,0,1,6.17-18.16c25.92-56.52,93-81.54,147.95-54.67S788.38,324.08,756.54,382.74Z"/></svg>
```

Comment:
Is the german content in this SVG supposed to be there? Also the identifier and data name are pretty wild. I would expect both to do something with "Cloudogu" and "logo".

Reply:
changed to "Cloudogu_Logo_<colorname>"